services/intersection-api/api/src/main/java/us/dot/its/jpo/ode/api/controllers/EventController.java

```diff
@@ -800,7 +800,6 @@ public ResponseEntity<List<SpatBroadcastRateEvent>> findSpatBroadcastRateEvents(
             Query query = spatBroadcastRateEventRepo.getQuery(intersectionID, startTime, endTime, latest);
             long count = spatBroadcastRateEventRepo.getQueryResultCount(query);
             logger.info("Returning SpatMinimumDataEventRepo Response with Size: " + count);
```

Comment:
polish(non-blocking): 
```suggestion
            logger.info("Returning SpatMinimumDataEventRepo Response with Size: {}", count);
```